.github/workflows/update-docs.yml

```diff
@@ -0,0 +1,47 @@
+name: Build and Deploy Gatsby Builder
```

Comment:
Seems like this should be something like "Deploy Component Library Docs"

Reply:
yep, that's a copy pasta error. Fixing!